stix/idl/io/stx_read_aux_fits.pro

```diff
@@ -0,0 +1,78 @@
+;+
+;
+; name:
+;       stx_read_aux_fits
+;
+; :description:
+;    Read the values contained in an auxiliary fits file and returns a correponding 'stx_aux_data' structure
+;
+;
+; :categories:
+;    fits, io
+;
+; :params:
+;    fits_path : in, required, type="string"
+;                the path of the FITS file to be read. Passed through to mrdfits.
+;
+;
+; :keywords:
+;
+;    primary_header : out, type="string array"
+;               an output float value;
```

Comment:
Not sure this description matches ;)

Reply:
I copied from the header stix/idl/processing/spectrogram/stx_read_pixel_data_fits_file.pro, since it is the same. We should change both then

Reply:
Ha ha hoist by my own petard! stx_read_spectrogram_fits_file.pro also has this issue. I did have the description I wanted to use under the following heading.

Reply:
;    primary_header : out, type="string array"
;              The header of the primary HDU of the pixel data files

This was what I meant to have.

Reply:
Correct it thanks!